### DISCUSSION TITLE
Mode VR

### DISCUSSION THREAD

**Discussion by MusicaFM** [gujjwal00/avnc#228] [Ideas]
Hello.  Can you add the option of virtual reality mode in a fixed and adjustable position in viewing and view separation to adjust the VR glasses?  thank you so much

**Comment by gujjwal00** [comment]
Unfortunately no. AVNC doesn't show any 3D content, so I am not sure how you expect it to work. Moreover, I have zero experience with VR development and don't have a VR device.
